exec/cnex/exclude.txt

```diff
@@ -55,11 +55,8 @@ net-test.neon              # net
 net-test-udp.neon          # net
 new-init-module.neon       # class
 new-init.neon              # class
-object.neon                # object
 object-isa.neon            # object
 object-isa-case.neon       # object
-object-null.neon           # object
-object-operator.neon       # object
 object-record.neon         # object
```

Comment:
What is is about this test case that doesn't run? It's no longer just "object", is it? What about the `object-isa` tests above?

Reply:
Actually, the Object-isa* tests run.  They have been removed.  Object-record fails for EQA, and has been noted.